src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  // height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $transition-time ease;
```

Comment:
The `$transition-time` variable is not defined in this file. Make sure to import the file where it is defined or define it in this file.